config/deploy.rb

```diff
@@ -21,11 +21,11 @@
 
 # Default value for :linked_files is []
 append  :linked_files,
-        'config/master.key',
```

Comment:
I think this might have been an accidental deletion?  We want to keep the `master.key` linked_file in this list.